Framework/Geometry/test/ObjectTest.h

```diff
@@ -859,6 +859,17 @@ class ObjectTest : public CxxTest::TestSuite {
                     expected, satol);
   }
 
+  void testVolumeUnitCube() {
+    Object_sptr cube = createUnitCube();
+    TS_ASSERT_DELTA(cube->volume(), 1.0, 1e-6)
+  }
+
+  void testVolumeSmallCappedCylinder() {
+    Object_sptr cylinder = createSmallCappedCylinder();
+    const double correctVolume = M_PI * 0.005 * 0.005 * 0.003;
+    TS_ASSERT_DELTA(cylinder->volume(), correctVolume, 1e-6)
+  }
+
```

Comment:
The test for the unit cube actually runs the Monte Carlo volume, but I'll add a more complex case just in case.